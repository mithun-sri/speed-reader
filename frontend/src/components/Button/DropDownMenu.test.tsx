import renderer, { ReactTestRendererJSON } from "react-test-renderer";
import {
  StyledFormControl,
  StyledInputLabel,
  StyledSelect,
} from "./DropDownMenu";

it("Matches snapshots", () => {
  const component1 = renderer.create(<StyledFormControl />);
  const tree1 = component1.toJSON() as ReactTestRendererJSON;
  expect(tree1).toMatchSnapshot();

  const component2 = renderer.create(<StyledInputLabel />);
  const tree2 = component2.toJSON() as ReactTestRendererJSON;
  expect(tree2).toMatchSnapshot();

  const component3 = renderer.create(<StyledSelect />);
  const tree3 = component3.toJSON() as ReactTestRendererJSON;
  expect(tree3).toMatchSnapshot();
});
