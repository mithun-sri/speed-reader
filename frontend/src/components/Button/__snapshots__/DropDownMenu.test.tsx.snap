// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshots 1`] = `
<div
  className="MuiFormControl-root css-zhwqls-MuiFormControl-root"
/>
`;

exports[`Matches snapshots 2`] = `
<label
  className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-animated css-1cwep4a-MuiFormLabel-root-MuiInputLabel-root"
/>
`;

exports[`Matches snapshots 3`] = `
<div
  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary css-1xetp59-MuiInputBase-root-MuiOutlinedInput-root-MuiSelect-root"
  onClick={[Function]}
>
  <div
    aria-controls=":r0:"
    aria-expanded="false"
    aria-haspopup="listbox"
    className="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input css-11u53oe-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onMouseDown={[Function]}
    role="combobox"
    tabIndex={0}
  >
    <span
      className="notranslate"
    >
      ​
    </span>
  </div>
  <input
    aria-hidden={true}
    aria-invalid={false}
    className="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
    onAnimationStart={[Function]}
    onChange={[Function]}
    tabIndex={-1}
  />
  <svg
    aria-hidden={true}
    className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
    data-testid="ArrowDropDownIcon"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M7 10l5 5 5-5z"
    />
  </svg>
  <fieldset
    aria-hidden={true}
    className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
  >
    <legend
      className="css-ihdtdm"
    >
      <span
        className="notranslate"
      >
        ​
      </span>
    </legend>
  </fieldset>
</div>
`;
